planner/core/integration_test.go

```diff
@@ -413,8 +415,12 @@ func (s *testIntegrationSerialSuite) TestBroadcastJoin(c *C) {
 	_, err := tk.Exec("explain select /*+ broadcast_join(fact_t, d1_t), broadcast_join_local(d1_t) */ count(*) from fact_t left join d1_t on fact_t.d1_k = d1_t.d1_k")
```

Comment:
why this query errors?

Reply:
casued by the hint?

Reply:
Yes, the local table of the left join must be the left table.